hack/conversion/conversion.go

```diff
@@ -55,7 +55,7 @@ func main() {
 	generator.OverwritePackage(*version, "")
 	generator.OverwritePackage("api", "newer")
 	for _, knownType := range kapi.Scheme.KnownTypes(*version) {
-		if strings.Contains(knownType.PkgPath(), "GoogleCloudPlatform/kubernetes") {
+		if strings.Contains(knownType.PkgPath(), "k8s.io/kubernetes") {
```

Comment:
> @smarterclayton that's my only doubt, is that leftover from @deads2k #4121 ?

Is this used by anything.  We use `cmd/genconversion/conversion.go` for generated conversions.